### PR TITLE
Remove the prometheus config map

### PR DESCRIPTION
**Pull request by andreaTP**

### CODE REVIEW

File: tools/cloudflow-operator/local-test/Running.md [hunk 1]
```diff
@@ -16,10 +16,9 @@ sbt -Ddocker.username=lightbend cloudflow-operator/docker:publish
 Install from helm chart using the docker image such as:
 ```
 helm upgrade -i cloudflow cloudflow-helm-charts/cloudflow \
-  --version "2.0.24" \
-  --set cloudflow_operator.image.name=lightbend/cloudflow-operator \
-  --set cloudflow_operator.image.tag=2.0.25-SNAP2-19-ff827053 \
-  --set cloudflow_operator.jvm.opts="-XX:MaxRAMPercentage=90.0 -XX:+UseContainerSupport" \
+  --version "2.0.26-RC15" \
+  --set cloudflow_operator.image.name=andreatp/cloudflow-operator \
```
Comment (RayRoestenburg) [inline tools/cloudflow-operator/local-test/Running.md:20]: andreatp? should be lightbend

File: tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/runner/SparkRunner.scala [hunk 2]
```diff
@@ -270,12 +261,11 @@ final class SparkRunner(sparkRunnerDefaults: SparkRunnerDefaults) extends Runner
...
           jmxExporterJar = "/prometheus/jmx_prometheus_javaagent.jar",
-          configFile = "/etc/cloudflow-runner/prometheus.yaml",
+          configFile = "/prometheus/prometheus.yaml",
           port = 2050))
       } else {
-        SparkApp.Monitoring(prometheus = SparkApp.Prometheus(
-          jmxExporterJar = agentPaths(PrometheusAgentKey),
-          configFile = PrometheusConfig.prometheusConfigPath(Runner.ConfigMapMountPath)))
+        SparkApp.Monitoring(prometheus = SparkApp
+          .Prometheus(jmxExporterJar = agentPaths(PrometheusAgentKey), configFile = "/prometheus/prometheus.yaml"))
```
Comment (RayRoestenburg) [inline tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/runner/SparkRunner.scala:268]: Do we still need agent paths?

Comment (RayRoestenburg) [reply tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/runner/SparkRunner.scala:268]: (It would be great if we could get rid of those)

Comment (RayRoestenburg) [reply tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/runner/SparkRunner.scala:268]: how does this work, pointing to `/prometheus/prometheus.yaml`?

**Review by RayRoestenburg — COMMENTED** [review]
LGTM got a few questions

File: tools/cloudflow-operator/local-test/Running.md (hunk 1 above)
Comment (andreaTP) [reply tools/cloudflow-operator/local-test/Running.md:20]: fixed

File: tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/runner/SparkRunner.scala (hunk 2 above)
Comment (andreaTP) [reply tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/runner/SparkRunner.scala:268]: I add a comment.

**Pull request merged**